### ISSUE TITLE
/sync events sometimes return an event missing channel.config.reactions field

### ISSUE THREAD

**Issue by linhewinner** [GetStream/stream-chat-swift#1862]
This is from our Sentry error log on our iOS client:

```
[ERROR] [com.apple.NSURLSession-delegate] [RequestDecoder.swift:75] [decodeRequestResponse(data:response:error:)] > keyNotFound(CodingKeys(stringValue: "reactions", intValue: nil), Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "events", intValue: nil), _JSONKey(stringValue: "Index 0", intValue: 0), CodingKeys(stringValue: "channel", intValue: nil), ChannelCodingKeys(stringValue: "config", intValue: nil)], debugDescription: "No value associated with key CodingKeys(stringValue: \"reactions\", intValue: nil) (\"reactions\").", underlyingError: nil))
```

It is not causing user visible bugs yet ... but this event is getting ignored by the client and I have no idea what it is and what the impact might be.
I want to create an issue to keep track of this. If there is a better backend repo where I should create the issue, plz let me know!

## GetStream Environment
**GetStream Chat version:** 4.12.0
**GetStream Chat frameworks:** StreamChat, StreamChatUI
**iOS version:**
**Swift version:**
**Xcode version:**
**Device:**

**Comment by tbarbugli** [comment]
This is related to #1835 @polqf good if you have a look

**Comment by tbarbugli** [comment]
this was resolved in 4.13.0
